Modules/Example/src/ExampleTask.cxx

```diff
@@ -48,6 +49,10 @@ void ExampleTask::initialize(o2::framework::InitContext& /*ctx*/)
 
   // Extendable axis
   mHistos[0]->SetCanExtend(TH1::kXaxis);
+
+  mCustomTH2F = new CustomTH2F("customTH2F");
+  getObjectsManager()->startPublishing(mCustomTH2F);
+  mCustomTH2F->SaveAs("custom.json");
```

Comment:
Is this needed? Someone might blindly copy this in their implementation.

Reply:
ah no, that was just to test